lib/core/utils/filter-html-attrs.js

```diff
@@ -2,25 +2,6 @@ import cache from '../base/cache';
 import getNodeAttributes from './get-node-attributes';
 import matchesSelector from './element-matches';
 
-/**
```

Comment:
Why was this file changed?

Reply:
I was confused at first too, but there's nothing special about the changes to this file. It refactors the same thing the rest of them do, removing this pattern:

```JavaScript
if (cache.get(outerHTML)) {
    node = cache.get(outerHTML);
  }
```

It looks strange because the first function wasn't the default export, so I had switched their order to comply with the new code standards.